index.html

```diff
@@ -8,8 +8,8 @@
     </head>
     <body>
         <header>
-            <div>    
-                <a id="logo" href="/index.html"><img src="img/logo.png"/></a>
+            <div>
+                <a id="logo" href="index.html"><img src="img/logo.png"/></a>
```

Comment:
This is not the best case to use ID, change the logic and use class instead. You can give className to the parent of logo, you can even remove parent div and use <a>